### DISCUSSION TITLE
Option "Login parameter nicht benutzen"

### DISCUSSION THREAD

**Discussion by rosti76** [SukramJ/aiohomematic#975] [Ideas]
Hallo,

Ich habe leider folgendes Problem - beim custom_homematic muss zwingend username/passwort angegeben werden.
Ich laufe derzeit mit ccu3 ohne zwingende Authentizierung, eingeschränkt auf IP-Adressen. Habe auch ein altes Modul, der keine Authentizierung gegen ccu3 unterstützt und stehe vor der Wahl, denn Ich kann HA nicht gleichzeitig mit dem anderem Modul einsetzen.

Wäre es möglich eine Option einzuführen, die bei Verbindungsaufbau die Authentizierung auslässt?

Danke.

lg

**Comment by SukramJ** [comment]
Nein, das ist möglich.
